app/views/attendance_mailer/flake.markerb

```diff
@@ -11,5 +11,5 @@ It'd be awesome if you were able to find another time that does work for you. Be
 
```

Comment:
<%= @tea_time.host.nickname %> here from Tea With Strangers. Just wanted to confirm that you canceled your seat at my tea time on <%= @tea_time.date_sans_year %>.

It's a shame that we won't get to share conversations, but you should try to get to another tea time. [See if something fits your schedule here](<%= url_for(controller: 'cities',
        action: 'show',
        id: @tea_time.city.city_code,
        only_path: false,
        utm_campaign: 'cancelspot',
        utm_medium: 'email',
        utm_content: 'show') %>).